src/java.base/share/man/java.md

```diff
@@ -3073,9 +3074,9 @@ The following items describe the syntax of `java` argument files:
 -   The argument file size must not exceed MAXINT (2,147,483,647) bytes.
 
 -   The launcher doesn't expand wildcards that are present within an argument
-    file. That means, an asterisk  `*` is passed on as-is to the starting VM.
+    file. That means an asterisk `*` is passed on as-is to the starting VM.
```

Comment:
```suggestion
    file. That means an asterisk (`*`) is passed on as-is to the starting VM.
```

Reply:
Addressed this and the other one in https://github.com/openjdk/jdk/pull/22236/commits/de0ac7c9676e405158c375f469ce1157bdc6f033